### DISCUSSION TITLE
2022-08-17 Weekly Meeting 69 Agenda - Web-based games & Interoperability

### DISCUSSION THREAD

**Discussion by antpb** [omigroup/omigroup#277] [Weekly Meetings]
Weekly Meeting 69 is happening on Wednesday August 17th, 2022 from 9:00 AM PST(16:00 UTC) - 10:00 AM PST (17:00 UTC)

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).
 
We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR/OMI Discord](https://discord.gg/NJtT9grz5E).
We regularly use the # omi-meeting-chat channel for text chat. 

## Who we are and why we're here:

The Open Metaverse Interoperability Group (OMI) is focused on bridging virtual worlds by designing and promoting protocols for identity, social graphs, inventory, and more. Our members include businesses and individuals working towards this common goal. 

If you’re an observer or here for the first time, this is a meeting where people who work on the OMI group initiatives discuss updates and contribution opportunities for current projects and goals. This is your chance to jump in and help!

We welcome you to introduce yourself in #omi-meeting-chat as we dive into the agenda items. There will be an open floor in the last 15 minutes of the meeting (agenda permitting) where we can discuss any items you feel are relevant to the current or near future OMI goals.

## Agenda
- Volunteer to take notes? Volunteer to record? 
- Welcome newcomers
- Check in on MSF

### Working Group Updates:
- [OMI Website update](https://github.com/omigroup/website)
- [Scripting group](https://github.com/omigroup/omi-scripting-group)
  - OMI developer hour
- [glTF Extensions Group](https://github.com/omigroup/gltf-extensions)
  - Progress on OMI_ref and authoring extensions
- [Media Group](https://github.com/omigroup/media)
- any others?

### Discussions: 
1) Web-based games and interoperability

![image](https://user-images.githubusercontent.com/8985705/184725567-25383ee2-e435-4f19-9b8c-3294c975dcc5.png)

### Future demo themes: (Vote now in the `#omi-vote` Discord channel!)
- `Wearables`
- `OMI-audio-emitter` exploration and demo 
- `OMI-collider` exploration and demo 
- `Avatars` Pt II
- `Portals` Pt II 
- `Generative and AI tooling`

**Comment by mrmetaverse** [comment]
# Website

**Reply by mrmetaverse** [reply]
add back to the readme, when do you meet? @Elirudite

**Comment by mrmetaverse** [comment]
# Scripting Group 
- New meeting time on Thursdays   
  - [view OMI calendar](https://calendar.google.com/calendar/u/0/embed?src=c_0dpugf9f83eq4qekmb6omqbjmg@group.calendar.google.com)

**Comment by mrmetaverse** [comment]
# Media group 
- All meetings are live now on discord and website calendar
  - 
![image](https://user-images.githubusercontent.com/63426722/185193816-00ec9dc1-f896-4694-8cec-fb9467d37410.png)
  - [Calendar](https://calendar.google.com/calendar/u/0/embed?src=c_0dpugf9f83eq4qekmb6omqbjmg@group.calendar.google.com)

**Comment by mrmetaverse** [comment]
# MSF Delegates
- Group meets at 15:00UTC every Wednesday (before OMI weekly) 
- Today group met to brainstorm on Miro, ways to refine and define how OMI and OMI-delegates want to participate in the MSF. 
- Ideating ways to be intentional about how we are passing on the information, and gathering feedback from our community. 
-  Fostering standards creation and policy making that are consistent with MSF and OMi goals, and challenging positions that are not consistent between OMI and MSF. 
- Meta discussions: where will we pass along info, who is taking notes, where and how do we document, etc. 
- The first layer addressed, convo will continue! 
- team added a few more channels to try and make it easier for our community: 
  - MSF links (miro board, processes, etc.) 
  - MSF opportunities/announcements channel 
- Want to know how to get involved with the Metaverse Standards Forum? Do you have ideas for how we can better serve you in MSF?  Reach out to our OMI MSF-delegates in discord. 
- Vote results: 
  -
![image](https://user-images.githubusercontent.com/63426722/185193153-223dbde3-0a6e-4f4e-a1f2-8fa1a2043902.png)
![image](https://user-images.githubusercontent.com/63426722/185193223-b5b881f8-fdd2-4a35-9dd0-35f0fef16c8f.png)

**Reply by mrmetaverse** [reply]
BEIAJ

**Reply by mrmetaverse** [reply]
categories may change over time, and a re-poll may be necessary

**Comment by mrmetaverse** [comment]
# Avatar-Interop (M3) 
- Group met yesterday in the M3 discord for a fun rig your own avatar workshop! 
- 25-30 attendees!
- [notes]()
- Get involved [here]()

**Comment by mrmetaverse** [comment]
# gLTF group 
- KHR Audio extension: pushing for more feedback from Khronos 
- presenting next week in front of Khronos 
- team to discuss what to present tomorrow during weekly meeting 
- talked a lot about physics materials, if you are interested JOIN US TOMORROW 
- @antpb gave a [great demo](https://twitter.com/antpb/status/1558526324460998656?s=20&t=Wph2rlzwwgXSvcIaQkSvdg) of his work with Wordpress gLTF CMS editor.

**Comment by mrmetaverse** [comment]
# UX Research Group pro-tem
- When do we take it vote in `omi-vote` channel on discord
- we need to confirm that there is availability of our community to regularly run this group
- the protocol labs proposal #222 is something the UX Research group can own/address. 
- Schedule a meeting next week for an info sesh to create a little executive summary working proposal #280

**Comment by mrmetaverse** [comment]
# DEMO time! `Web-based-games`
![image](https://user-images.githubusercontent.com/63426722/185196498-0a3d1d7d-21cf-470a-8aba-5a3ce0a44eb8.png)

## Limbofeather with ZestyMarket
- a lot of the webxr experiences visible on the Meta Quest browser have the Zesty SDK installed 
- ![image](https://user-images.githubusercontent.com/63426722/185196770-facc99c2-0021-4fe3-b94a-b5fbe1153797.png)
- seems to indicate an increase in VR adoption 
- ![image](https://user-images.githubusercontent.com/63426722/185196818-569d9e8d-7bd6-4795-9628-e090d6dc214b.png)
- ![image](https://user-images.githubusercontent.com/63426722/185196857-b54ebdd8-66d9-43c4-ac69-92cec6eb7dcb.png)
- They help creators monetize their webxr work. 
- Zesty is now developing their own web based game as well, a Multiplayer First-person shooter! 
- They found that when people browse webxr, there are usually multiple people on a zesty market site at any given time coming from multiple headsets. 
- There hasn't been many webxr multiplayer games. 
- ![image](https://user-images.githubusercontent.com/63426722/185198080-137341a4-dbdc-4a57-840c-87dbc9859754.png)
- Wanted to make something fun, and iteratively develop. 
- ![image](https://user-images.githubusercontent.com/63426722/185197306-a0db3efc-b0b9-4dc1-be9c-86a12f2f7452.png)
- multiplayer works
- ![image](https://user-images.githubusercontent.com/63426722/185197378-8f1af2e2-c552-4e76-85aa-41b93970e1df.png)
- fun little jump pads 
- ![image](https://user-images.githubusercontent.com/63426722/185197414-8ec7991e-41aa-4282-8c28-e24812ac0b3f.png)
- try out the game here: https://rr.zesty.xyz
- Voice chat in the works. 
- when you lose all your health, you respawn away from your opponent. 
- ![image](https://user-images.githubusercontent.com/63426722/185199583-afe206b7-abd0-42d4-9deb-101f1988f3d7.png)
- about 1000 users so far in 2 days since launching alpha. Its been exciting seeing strangers from the web seeing our banner ad and visiting our little demo game. 
- working on adding better support for voice, updates on arena activity, health, power-ups, weapons, maps, and more. 
- we love how webxr allows people to meet each other collaboratively 
- has hand support too! (pinch to shoot for now)
  - ![image](https://user-images.githubusercontent.com/63426722/185199956-7f97daf1-ca60-4bae-bad0-90e0444e0f3f.png)
- Questions: 
  - This may be out of scope for you all, but would it be possible to make this cross-platform so you can play with people on a browser on PC?
    - yeah we are working on that now! Adding WASD support, shooting with remotes instead of controllers, etc. 
  - Why is Tuesday such a hit on your metrics? 
    - Probably because when the quest headset increased their prices, there was a surge in the run up before the price increase, and post price increase we say a drop off again. 
  - What are you looking to solve and do with your analytics data? What has been most challenging?
    - One thing that has been interesting to learn about is that 3 days ago Pico 4 started making more announcements around their new hardware. Our beacon-feed channel in the zesty discord shows which headsets are getting the most traffic. 
      - ![image](https://user-images.githubusercontent.com/63426722/185198956-57130773-46f3-4f34-b4e8-71fea01dbcf0.png)
      - we found that it was the new PICO 4 that was showing up as a new type of headset. 
      - Every night we get more info from the SDK beacon. 
      - We don't collect any sort of IP or PII (personally identifiable information) 
- Want to get involved? Hop into the [Zesty Market Discord](https://discord.gg/eVCwdCqrFQ) and introduce yourself.

**Comment by mrmetaverse** [comment]
# Next weeks topic 
- vote in our `omi-vote` discord channel 
- Topics: 
  - avatars pt. ii 
  - portals pt. ii 
  - wearables
  - using OMI draft protocols

**Reply by mrmetaverse** [reply]
Of 11 responses we have a tie.. 
![Screenshot_20220818-230102.png](https://user-images.githubusercontent.com/63426722/185533753-307b4d5e-7155-4115-80a6-ba71a82e7983.png)

Next theme is Avatars and/or portals pt. ii

**Comment by futuristudios** [comment]
conversation was video recorded. here's the [file on ipfs](https://ipfs.io/ipfs/QmdUDyhpsVhfoHzfhKzG87G3PyMMpBgy3nonNJnXHAoGsp), could someone with permissions please uploaded to [OMI youtube channel](https://www.youtube.com/channel/UCB3huP95JrOTma5LfiDCd4A).

**Reply by mrmetaverse** [reply]
Thank you!! Cc: @LightLodges and @pcsmgnt I will create a ticket and can upload.
